src/Parlot/Properties/AssemblyInfo.cs

```diff
@@ -1,3 +1,3 @@
-#if !NET8_0_OR_GREATER
```

Comment:
This should be always "on" with pre-compiler directives.

should add to csproj:

```
<PackageReference Include="PolySharp" Version="1.14.1" PrivateAssets="all" />
```
then remove files `IsExternalInit.cs` and `SkiptLocalsInitAttribute.cs`